### PR TITLE
feat: added util transformToNullableBoolean

### PR DESCRIPTION
**Pull request by mansurishmuhametov**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [x] The commit message follows our guidelines: https://github.com/ngxs/store/blob/master/CONTRIBUTING.md#commit
-   [x] Tests for the changes have been added (for bug fixes / features)
-   [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: packages/cdk/integration/tests/jest/class-transformer/transform-to-boolean.spec.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { transformToBoolean } from '@angular-ru/cdk/class-transformer';
+import { TransformFnParams } from 'class-transformer';
+
+describe('[TEST] TransformToBoolean', () => {
+    it('should returne true', () => {
```
Comment (splincode) [inline packages/cdk/integration/tests/jest/class-transformer/transform-to-boolean.spec.ts:5]: `return`

File: packages/cdk/integration/tests/jest/class-transformer/transform-to-boolean.spec.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            { value: '' } as TransformFnParams,
+            { value: '   ' } as TransformFnParams
+        ];
+
+        validValues.forEach((item: TransformFnParams): void => {
+            expect(transformToBoolean(item)).toBe(true);
+        });
+    });
+
+    it('should returne false', () => {
```
Comment (splincode) [inline packages/cdk/integration/tests/jest/class-transformer/transform-to-boolean.spec.ts:23]: `return`

File: packages/cdk/integration/tests/jest/class-transformer/transform-to-boolean.spec.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
+import { transformToBoolean } from '@angular-ru/cdk/class-transformer';
+import { TransformFnParams } from 'class-transformer';
+
+describe('[TEST] TransformToBoolean', () => {
+    it('should returne true', () => {
+        const validValues: TransformFnParams[] = [
```
Comment (splincode) [inline packages/cdk/integration/tests/jest/class-transformer/transform-to-boolean.spec.ts:6]: `validValues` -> `data`

File: packages/cdk/integration/tests/jest/class-transformer/transform-to-boolean.spec.ts [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+            { value: '   ' } as TransformFnParams
+        ];
+
+        validValues.forEach((item: TransformFnParams): void => {
+            expect(transformToBoolean(item)).toBe(true);
+        });
+    });
+
+    it('should returne false', () => {
+        const validValues: TransformFnParams[] = [
```
Comment (splincode) [inline packages/cdk/integration/tests/jest/class-transformer/transform-to-boolean.spec.ts:24]: `validValues` -> `data`

File: packages/cdk/integration/tests/jest/class-transformer/transform-to-nullable-boolean.spec.ts [hunk 5]
```diff
@@ -0,0 +1,46 @@
+import { transformToNullableBoolean } from '@angular-ru/cdk/class-transformer';
+import { TransformFnParams } from 'class-transformer';
+
+describe('[TEST] TransformToNullableBoolean', () => {
+    it('should returne true', () => {
```
Comment (splincode) [inline packages/cdk/integration/tests/jest/class-transformer/transform-to-nullable-boolean.spec.ts:5]: `return`

File: packages/cdk/integration/tests/jest/class-transformer/transform-to-nullable-boolean.spec.ts [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+        const validValues: TransformFnParams[] = [
```
Comment (splincode) [inline packages/cdk/integration/tests/jest/class-transformer/transform-to-nullable-boolean.spec.ts:6]: `validValues` -> `data`

File: packages/cdk/integration/tests/jest/class-transformer/transform-to-nullable-boolean.spec.ts [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+            { value: '   ' } as TransformFnParams
+        ];
+
+        validValues.forEach((item: TransformFnParams): void => {
+            expect(transformToNullableBoolean(item)).toBe(true);
+        });
+    });
+
+    it('should returne false', () => {
+        const validValues: TransformFnParams[] = [
```
Comment (splincode) [inline packages/cdk/integration/tests/jest/class-transformer/transform-to-nullable-boolean.spec.ts:24]: `validValues` -> `data`

File: packages/cdk/integration/tests/jest/class-transformer/transform-to-nullable-boolean.spec.ts [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+            { value: false } as TransformFnParams,
+            { value: 0 } as TransformFnParams
+        ];
+
+        validValues.forEach((item: TransformFnParams): void => {
+            expect(transformToNullableBoolean(item)).toBe(false);
+        });
+    });
+
+    it('should returne null', () => {
```
Comment (splincode) [inline packages/cdk/integration/tests/jest/class-transformer/transform-to-nullable-boolean.spec.ts:36]: `return`

**Review by splincode — CHANGES_REQUESTED** [review]

File: packages/cdk/integration/tests/jest/class-transformer/transform-to-boolean.spec.ts (hunk 1 above)
Comment (mansurishmuhametov) [reply packages/cdk/integration/tests/jest/class-transformer/transform-to-boolean.spec.ts]: fixed that

File: packages/cdk/integration/tests/jest/class-transformer/transform-to-boolean.spec.ts (hunk 2 above)
Comment (mansurishmuhametov) [reply packages/cdk/integration/tests/jest/class-transformer/transform-to-boolean.spec.ts]: fixed that

File: packages/cdk/integration/tests/jest/class-transformer/transform-to-boolean.spec.ts (hunk 3 above)
Comment (mansurishmuhametov) [reply packages/cdk/integration/tests/jest/class-transformer/transform-to-boolean.spec.ts]: good name

File: packages/cdk/integration/tests/jest/class-transformer/transform-to-boolean.spec.ts (hunk 4 above)
Comment (mansurishmuhametov) [reply packages/cdk/integration/tests/jest/class-transformer/transform-to-boolean.spec.ts]: good name

File: packages/cdk/integration/tests/jest/class-transformer/transform-to-nullable-boolean.spec.ts (hunk 5 above)
Comment (mansurishmuhametov) [reply packages/cdk/integration/tests/jest/class-transformer/transform-to-nullable-boolean.spec.ts]: +

File: packages/cdk/integration/tests/jest/class-transformer/transform-to-nullable-boolean.spec.ts (hunk 6 above)
Comment (mansurishmuhametov) [reply packages/cdk/integration/tests/jest/class-transformer/transform-to-nullable-boolean.spec.ts]: good

File: packages/cdk/integration/tests/jest/class-transformer/transform-to-nullable-boolean.spec.ts (hunk 7 above)
Comment (mansurishmuhametov) [reply packages/cdk/integration/tests/jest/class-transformer/transform-to-nullable-boolean.spec.ts]: changed

File: packages/cdk/integration/tests/jest/class-transformer/transform-to-nullable-boolean.spec.ts (hunk 8 above)
Comment (mansurishmuhametov) [reply packages/cdk/integration/tests/jest/class-transformer/transform-to-nullable-boolean.spec.ts]: fixed

**Review by splincode — APPROVED** [review]

**Pull request merged**